src/author.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'item'
+
+class Author
+  attr_accessor :first_name, :last_name
+
+  @items = []
+
+  def initialize(first_name, last_name)
+    self.class.id_counter = (self.class.id_counter || 0) + 1
+    @first_name = first_name
+    @last_name = last_name
+    @id = self.class.id_counter
+  end
+
+  attr_reader :id, :items
+
```

Comment:
Looks like `:items` is a private instance variable according to the table below, so I think we should remove it from the attr_reader, but let me know if this would not affect your functionality. If removing this reader will affect how your code works then keep it.

![image](https://github.com/Diegogagan2587/catalog-of-my-things/assets/61764778/cbdfc7e2-79e9-48f0-81f1-1c8fcbfeeaaf)